### ISSUE TITLE
Config assumes M1 Mac

### ISSUE THREAD

**Issue by JLLeitschuh** [guylabs/macstrap-config#4]
If you attempt to install rosetta on a non-M1 mac, the config fails.

https://github.com/guylabs/macstrap-config/blob/0a1b7d18717fa7da1ab3bd2585e3d94b84228a11/custom-scripts/pre-install-0-setup.sh#L35-L36

Additionally, this line also assumes an M1 mac book pro: `eval "$(/opt/homebrew/bin/brew shellenv)"`

https://github.com/guylabs/macstrap-config/blob/9e033771b67212c03518da54b8e1c0dbfa2039ed/configs/dotfiles/.zshrc#L64-L65

**Comment by JLLeitschuh** [comment]
The only two locations currently causing issues are the locations listed above. Other places in the code seem to attempt to mitigate this. Could something similar be used?

https://github.com/guylabs/macstrap-config/blob/895c73957ce52b2c9f15c93dcdc1c3349aaba91e/commands/install.sh#L5-L11

**Comment by guylabs** [comment]
Yes we can use that. Feel free to create a pull request with your state what works on x86. That would be great!
